azure-pipelines.yml

```diff
@@ -206,6 +206,7 @@ stages:
           include:
           - src/Compilers/*
           - src/Dependencies/*
+          - eng/*
```

Comment:
seems like we should run compiler CI when build scripts change

Reply:
Since you are touching this, could you also fix https://github.com/dotnet/roslyn/issues/65602 in this PR?

Reply:
Added the compiler generator tools path - let me know if there were other directories you were thinking of